tasks/build-burp.yml

```diff
@@ -11,7 +11,16 @@
   package:
     name: "{{ build_dependencies | join(',') }}"
     state: present
+  when: ( ansible_os_family != 'FreeBSD' and not burp_freebsd_use_portinstall )
```

Comment:
This conditional is breaking linux builds, as you can see here:
https://travis-ci.org/CoffeeITWorks/ansible_burp2_client/builds/340463642#L4015 

will merge into branch feature/13_freebsd_port and fix for #13